### PR TITLE
fix: carbon components overrides

### PR DESCRIPTION
**Pull request by mamartinezmejia**

### CODE REVIEW

**Comment by MCatherine1994** [comment]
oh the "Fix" needs to be lower case "fix", otherwise release-please can't recognize it

File: style-sheets/carbon-components-overrides.scss [hunk 1]
```diff
@@ -10,6 +14,14 @@
   $button-border-radius: 0.25rem
 );
 
+@use '@carbon/styles/scss/grid';
+
+@include grid.flex-grid();
+
+body {
+  font-family: map.get(typeFamily.$type-family, 'sans');
```
Comment (MCatherine1994) [inline style-sheets/carbon-components-overrides.scss:22]: oh Maria @mamartinezmejia, you want to test this locally first, previous we're getting some errors, refers this issue: https://github.com/bcgov/nr-fsa-theme/issues/34. make sure it works fine locally

Comment (mamartinezmejia) [reply style-sheets/carbon-components-overrides.scss:22]: Oh, yes, we did it. Thanks for checking.

File: style-sheets/carbon-components-overrides.scss [hunk 2]
```diff
@@ -10,6 +12,14 @@
   $button-border-radius: 0.25rem
 );
 
+@use '@carbon/styles/scss/grid';
+
+@include grid.flex-grid();
+
+body {
+  font-family: ‘BCSans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
```
Comment (MCatherine1994) [inline style-sheets/carbon-components-overrides.scss:20]: I won't recommend we do any hardcode style here, so another way to update the font family is to remove it from this file, and make the change in your project. So in your project, import the font family design token and make the changes you want.

Comment (craigyu) [reply style-sheets/carbon-components-overrides.scss:20]: On that note can we change the single quote from `‘` `’` to `'`?
image

**Comment by craigyu** [comment]
What are the objectives of this PR? Are the added styles for everyone or just team alliance? 

Ideally I would like to keep this file as it is so SPAR does not have to do a lot of work to refactor styling on every page.

Here's a few examples on how the current changes affect SPAR:

- the second login button is bigger 
    - image
- the margin is huge for dashboard
    - Screenshot 2023-07-17 at 2 46 57 PM
- spaced out icons
    - image
 

cc: @MCatherine1994

File: style-sheets/carbon-components-overrides.scss (hunk 2 above)
Comment (mamartinezmejia) [reply style-sheets/carbon-components-overrides.scss]: Thanks, Craig!

Comment (mamartinezmejia) [reply style-sheets/carbon-components-overrides.scss]: Hi @MCatherine1994 

As per our agreement, this part will be part of the customs styles as the given path for the fonts doesn't work for all teams

**Review by craigyu — APPROVED** [review]
Looks good!

**Pull request merged**